src/main/scripts/build.js

```diff
@@ -61,6 +61,12 @@ if (! validator(registry)) {
   throw "Registry fails validation";
 };
 
+/* check for duplicate codes */
+
+const dups = [...new Set(registry.map(i => i.code).filter((v,i,a) => a.indexOf(v) !== i))]
```

Comment:
Couldn't duplicate checking be accomplished by changing the `>` on line 73 with a `>=`?

Reply:
Well the error message would need a conditional perhaps to make more sense. As is, it lists all duplicates.

Reply:
Different error message is fine. I suggest not having two different functions essentially do the same thing.